docs/GettingStartedDocs/install_host_verify_Windows.md

```diff
@@ -0,0 +1,78 @@
+# Install the Open Enclave Host-Verify SDK NuGet Package
+
+## Platform requirements
+
+- Windows Server 2016 or 2019
+
+## Software Prerequisites
+
+### Microsoft Visual Studio Build Tools 2019
+
+Install [Visual Studio Build Tools 2019](https://aka.ms/vs/16/release/vs_buildtools.exe). Choose the "C++ build tools" workload. Visual Studio Build Tools 2019 has support for CMake Version 3.15 (CMake ver 3.12 or above is required for building Open Enclave Host-Verify SDK). For more information about CMake support, look [here](https://blogs.msdn.microsoft.com/vcblog/2016/10/05/cmake-support-in-visual-studio/).
+
+### Git for Windows 64-bit
+
+Download [Git for Windows 64-bit](https://git-scm.com/download/win).
+
+Install Git and add Git Bash to the PATH environment variable.
+Typically, Git Bash is located in `C:\Program Files\Git\bin`.
+Currently the Open Enclave Host-Verify SDK build system uses bash scripts to configure
+and build Linux-based 3rd-party libraries.
+
+Open a command prompt and ensure that Git Bash is added to PATH.
+
+```cmd
+C:\>where bash
+C:\Program Files\Git\bin\bash.exe
+```
+
+Tools available in the Git bash environment are also used for test and sample
+builds. For example, OpenSSL is used to generate test certificates, so it is
+also useful to have the `Git\mingw64\bin` folder added to PATH. This can be checked
+from the command prompt as well:
+
+```cmd
+C:\>where openssl
+C:\Program Files\Git\mingw64\bin\openssl.exe
+```
+
+### Clang
+
+Download [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe).
+Install Clang 7.0.1 and add the LLVM folder (typically C:\Program Files\LLVM\bin)
+to PATH. Open Enclave Host-Verify SDK uses clang to build the enclave binaries.
+
+Open up a command prompt and ensure that clang is added to PATH.
+
+```cmd
+C:\> where clang
+C:\Program Files\LLVM\bin\clang.exe
+C:\> where llvm-ar
+C:\Program Files\LLVM\bin\llvm-ar.exe
+C:\> where ld.lld
+C:\Program Files\LLVM\bin\ld.lld.exe
+```
+
+## Download and install the Open Enclave Host-Verify SDK NuGet Package
+
+Download the required Windows NuGet Package from [here](https://github.com/openenclave/openenclave/releases) and place it in a directory of your choice. Use the command below to install the NuGet package. In this example, we are placing the NuGet Package in `C:\openenclave_nuget` and installing it to `C:\oe`.
```

Comment:
I tested it in a command line interface, so that's why I followed the instructions in the following page to generate it:
 - https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Windows.md

Reply:
Can you test it with the VS manage nuget package approach that is consistent with other OE developer docs I referenced?

Reply:
Sorry, I'm kind of confused: we were able to locate the NuGet package using "Manage NuGet Packages..." but open-enclave.OEHOSTVERIFY does not require enclaves. Therefore, I may be able to briefly mention such method on `VisualStudioLinux.md` and `VisualStudioWindows.md`, but what should I add on `visualstudio_dev.md`? Like adding a new section for links pointing to the `install_*.md` pages (but it seems unrelated to the Visual Studio part)?